src/containers/App.test.js

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { render } from "@testing-library/react";
 import App from "./App";
 
-test("renders learn react link", () => {
+test("renders correctly whithout crash", () => {
```

Comment:
i think you can just delete this file, we don't want random code like this lying around